vault/logical_system_activity.go

```diff
@@ -271,18 +288,36 @@ func parseStartEndTimes(d *framework.FieldData, billingStartTime time.Time) (tim
 		endTime = endTime.UTC()
 	}
 
+	isAlignedTime := false
```

Comment:
We don't want to modify parseStartEndTimes as it is common to counters and export api. I am not sure if it is being used by something else to. What we can do is first get the start and endtime from this function and then call another function named getBillingPeriodTime or something to get the billing period start and end date and use that in the counters api.